ChatApp/Extensions/UIColor+Extension.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  UIColor+Extension.swift
+//  chatApp.test
+//
+//  Created by Ana Dzebniauri on 19.04.23.
+//
+
+import UIKit
+
+extension UIColor {
+    convenience init(red: Int, green: Int, blue: Int, alpha: CGFloat) {
+        self.init(
+            red: CGFloat(red)/255,
+            green: CGFloat(green)/255,
+            blue: CGFloat(blue)/255,
+            alpha: alpha
+        )
+    }
+    
+    //MARK: - Typing Area Colors
+    static let messageTextColor = UIColor(red: 25, green: 25, blue: 25, alpha: 1)
+    static let SystemGreyTextColor = UIColor(red: 199, green: 199, blue: 199, alpha: 1)
+    static let typeAreaBoaderColor = UIColor(red: 159, green: 96, blue: 255, alpha: 1)
+
+    //MARK: - Bubble Colors
+    static let recipientColor = UIColor(red: 241, green: 241, blue: 241, alpha: 1)
+    static let senderColor = UIColor(red: 218, green: 195, blue: 255, alpha: 1)
```

Comment:
here also having recipienColor in UIColors extension does not seems right. you can have here some specific colors with clear name like darkRed = ..., lightBlue or smt like this, but having senderColor which only can be used for senders component is not right.